yoyodyne/models/lstm.py

```diff
@@ -199,6 +199,7 @@ def decode(
         encoder_mask: torch.Tensor,
         encoder_out: torch.Tensor,
         target: Optional[torch.Tensor] = None,
+        teacher_forcing: bool = True,
```

Comment:
Why can't this switch on whether or not `target` is null? The documentation for `target` seems to suggest that's what it does.